public/general_components/service_card/plugin_card.tsx

```diff
@@ -37,7 +37,7 @@ export const registerPluginCard = (
           }}
         >
           {i18n.translate('flowFrameworkDashboards.opensearchFlowCard.footer', {
-            defaultMessage: 'Try OpenSearch Flow',
+            defaultMessage: 'Try AI Search Flows',
```

Comment:
Can you replace with the constant `PLUGIN_NAME`? Then this change won't be necessary in the future if it updates again.